connect/resource/template.py

```diff
@@ -27,7 +29,9 @@ def render(self, pk, request_id):
         return ActivationTileResponse(response)
 
     def get(self, pk):
+        # type: (str) -> ActivationTemplateResponse
         return ActivationTemplateResponse(template_id=pk)
 
     def list(self):
+        # type: () -> Any
```

Comment:
more or less pointless type-hinting (there and in similar places)

Reply:
True. I'll check those and fix them. This one should probably be `# type: () -> List[BaseModel]` on BaseResource, and a list of specific BaseModel subclasses on derived resources.

Reply:
Done.